### ISSUE TITLE
Add reward eligibility check to reward card & cover "specific team" use case

### ISSUE THREAD

**Issue by JonRay15** [vegaprotocol/frontend-monorepo#5618]
## Story

As a user
I want to know whether I am eligbible for a reward 
So that I know whether to try to compete to earn it

## Acceptance Criteria

- [x] When my wallet is connected, on the reward card it should show along the bottom:
  - [x] Check my entity scope to see if I am eligible (use the teams API to see if I am in a team or not) and display tick / cross (see additional info)
  - [x] Check my staked vega balance and compare to the value on the reward, if I have enough tick, if not cross 
  - [ ] Check my average position as found on [slack thread hopefully]
  - [ ] In each of the above cases if the requirement is zero then show the tick
  - [x] Each of the three requirements should have their own tick/cross
 - [x] Fix alignment of requirement titles and icons (https://vegaprotocol.slack.com/archives/C9E6XA8GK/p1705063617891869), if I am meeting it tick if not cross
 - [ ] Note that we're not sure where to get the average  notional currently ...if we dont have this, we will advise that any rewards created dont have this for now, and therefore if we see a zero we can apply the green tick

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

## Sketch

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/f7ca0ed6-db46-4b6c-b173-8b60394792d9)

The standard tick / cross will do .... **make them vega green / red**

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/2e43d2e3-1b29-4f24-95a7-0678ba62246b)

Suggest adding this icon from foundations to the existing team icon, probably dont need the little vegabond

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/281b0b4d-ad87-46e3-907c-c22cbcec27d3)

## Additional details / background info

This is how the eligibility currently looks which is wrong.

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/c6ba4fdd-7eec-4094-9e64-382cc3098342)

This should simply say "Eligible" and then the options should be:

- All teams, eg. "entityScope": "ENTITY_SCOPE_TEAMS" and "teamScope": null 
- Listed teams, eg. "entityScope": "ENTITY_SCOPE_TEAMS" and "teamScope": some list of team IDs (display team names on hover)
- All individuals, eg. "entityScope": "ENTITY_SCOPE_INDIVIDUALS" and "individualScope": "INDIVIDUAL_SCOPE_UNSPECIFIED"
- Team individuals, eg. "entityScope": "ENTITY_SCOPE_INDIVIDUALS" and "individualScope": "INDIVIDUAL_SCOPE_IN_TEAM"
- Solo individuals, eg. "entityScope": "ENTITY_SCOPE_INDIVIDUALS" and "individualScope": "INDIVIDUAL_SCOPE_NOT_IN_TEAM"

**Comment by JonRay15** [comment]
This probably needs to be put on hold until the changes in #5580 have been done since it may change what we want to do here

**Comment by JonRay15** [comment]
I take back my earlier statement ... this isnt blocked by #5580 and we need to sort it out quite soon.

**Comment by JonRay15** [comment]
Core ticket here for notional time weighted position 

https://github.com/vegaprotocol/vega/issues/10831

**Comment by JonRay15** [comment]
This Core ticket should be merged soon per Gordon
